src/frontend/screens/Settings/ProjectConfig/index.tsx

```diff
@@ -44,17 +45,22 @@ const m = defineMessages({
 
 export const ProjectConfig = () => {
   const { formatMessage: t } = useIntl();
+  // TODO: dummy state, mostly for demonstrative purposes
+  const [role] = React.useState<"participant" | "coordinator">("participant");
```

Comment:
highlighting this since I'm guessing this would be handled in some more appropriate way at some point